InternxtDesktop/AppDelegate.swift

```diff
@@ -172,13 +174,45 @@ class AppDelegate: NSObject, NSApplicationDelegate {
         
     }
     
-    func loginSuccess() {
+    private func refreshTokens() {
+        Task{
+            self.logger.info("Refreshing tokens")
+            do {
+                try await authManager.refreshTokens()
+                self.logger.info("Tokens refreshed correctly")
+            } catch{
+                AuthError.UnableToRefreshToken.reportToSentry()
+                guard let apiClientError = error as? APIClientError else {
+                    return
+                }
+                
+                // 401 means token is expired, so we logout the user
```

Comment:
A variable that explains this could be better than a comment, which is usually not maintained as it does not affect the behavior of the codebase. 

`let expiredToken = apiClientError.statusCode == 401`